client/src/components/Navbar/Menus/Authenticated/Desktop.tsx

```diff
@@ -0,0 +1,56 @@
+import { Grid, makeStyles, Typography, Avatar } from '@material-ui/core';
+import { AiOutlineCaretDown } from 'react-icons/ai';
+import { useHistory } from 'react-router';
+
+const useStyles = makeStyles({
+  root: {
+    width: '60%',
+    maxWidth: '600px',
+  },
+  links: {
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  text: {
+    color: 'white',
+    fontWeight: 'bolder',
+    fontSize: 16,
+  },
+  account: {
+    width: '100%',
+    justifyContent: 'space-evenly',
+    alignItems: 'center',
+  },
+});
+
+export const Desktop = (): JSX.Element => {
+  const classes = useStyles();
+  const history = useHistory();
+
+  return (
+    <Grid item className={classes.root}>
+      <Grid container className={classes.links} spacing={0}>
+        <Grid item onClick={() => history.push('/discover')}>
+          <Typography className={classes.text}>Discover</Typography>
+        </Grid>
+        <Grid item onClick={() => history.push('/messages')}>
+          <Typography className={classes.text}>Messages</Typography>
+        </Grid>
+        <Grid item onClick={() => history.push('/notifications')}>
+          <Typography className={classes.text}>Notifications</Typography>
+        </Grid>
```

Comment:
Can you make these links or possibly buttons so they have more intuitive hover style?

Reply:
I'm having a visual bug with the display when I changed it to a `<Link>` where the [font-family and weight appear to change](https://www.loom.com/share/ea9db718c6ba4e7caaa5760fffeb25d8) upon click and reset upon refresh.